cloudwatchlogs/cloudwatchlogs_lambda.js

```diff
@@ -25,31 +25,48 @@ exports.handler = function(event, context) {
 				context.succeed("Success");
 			}
 
-			var req = https.request(options, function(res) {
-				var body = '';
-				console.log('Status:', res.statusCode);
-				res.setEncoding('utf8');
-				res.on('data', function(chunk) { body += chunk; });
-				res.on('end', function() {
-					console.log('Successfully processed HTTPS response');
-					context.succeed(); });
-			});
-
-			req.on('error', context.fail);
+			var requestsSent = 0;
+			var requestsFailed = 0;
+			var finalizeContext = function() {
+				var tot = requestsSent + requestsFailed;
+				if (tot == awslogsData.logEvents.length) {
+					if (requestsFailed > 0) {
+						context.fail();
```

Comment:
`context.fail` and `context.succeed` allow to pass in a string value. Changing your code to this might be a nice optional addition:

```js
if (requestsFailed > 0) {
  context.fail(requestsFailed + " / " + requestsSent " events failed");
} else {
  context.succeed(requestsSent + " requests sent");
}
```